src/status_im/ui/screens/intro/views.cljs

```diff
@@ -545,14 +545,11 @@
 (defview wizard-enter-phrase []
   (letsubs [wizard-state [:intro-wizard/enter-phrase]]
     [react/keyboard-avoiding-view {:style {:flex 1}}
-     [toolbar/toolbar
-      {:style {:border-bottom-width 0
-               :margin-top 16}}
+     [toolbar/toolbar {:style {:border-bottom-width 0}}
       (toolbar/nav-button
        (actions/back #(re-frame/dispatch [:intro-wizard/navigate-back])))
       nil]
-     [react/view {:style {:flex 1
-                          :justify-content :space-between}}
+     [react/view {:style {:flex 1 :justify-content :space-between}}
```

Comment:
What's the point of putting of all these properties to one line? It makes it harder to read imo